### DISCUSSION TITLE
Just a suggestion

### DISCUSSION THREAD

**Discussion by Unknownboiiiiii** [aniyomiorg/aniyomi#583] [Ideas]
Added a feature where if you watch any anime/read any manga and forgot to favorite it and also forgot to check what's the name of that anime/manga it will show which animes/mangas you watched/read recently

**Comment by sinnghabhay80** [comment]
History is there already.

**Reply by Unknownboiiiiii** [reply]
Where?

**Reply by Dream3955** [reply]
More -> History

**Reply by Unknownboiiiiii** [reply]
K thanks
